### ISSUE TITLE
Getting error on Exception: database table is locked

### ISSUE THREAD

**Issue by winpaul** [sj26/mailcatcher#228]
Getting below error after first mailcatch:

```
Exception: database table is locked
    Backtrace:
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/sqlite3-1.3.10/lib/sqlite3/statement.rb:67:in `step'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/sqlite3-1.3.10/lib/sqlite3/statement.rb:67:in `execute'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/mailcatcher-0.6.1/lib/mail_catcher/mail.rb:44:in `add_message'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/mailcatcher-0.6.1/lib/mail_catcher/smtp.rb:46:in `receive_message'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/smtpserver.rb:539:in `process_data_line'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/smtpserver.rb:196:in `receive_line'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:64:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/em/protocols/linetext2.rb:65:in `receive_data'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/eventmachine.rb:187:in `run_machine'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/eventmachine-1.0.5/lib/eventmachine.rb:187:in `run'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/mailcatcher-0.6.1/lib/mail_catcher.rb:155:in `run!'
       /usr/local/rvm/gems/ruby-1.9.3-p547/gems/mailcatcher-0.6.1/bin/mailcatcher:5:in `<top (required)>'
       /usr/local/rvm/gems/ruby-1.9.3-p547/bin/mailcatcher:23:in `load'
       /usr/local/rvm/gems/ruby-1.9.3-p547/bin/mailcatcher:23:in `<main>'
       /usr/local/rvm/gems/ruby-1.9.3-p547/bin/ruby_executable_hooks:15:in `eval'
       /usr/local/rvm/gems/ruby-1.9.3-p547/bin/ruby_executable_hooks:15:in `<main>'
    Please submit this as an issue at http://github.com/sj26/mailcatcher/issues
```

**Comment by sj26** [comment]
This seems to pop up on some systems, and we haven't identified why. What operating system / ruby version / sqlite version is your system running?

**Comment by sj26** [comment]
Let's fix this by removing sqlite in #298.
